src/runner.js

```diff
@@ -61,6 +65,24 @@ assign(Runner.prototype, {
 
   },
 
+  _run: function() {
+    var runner = this
+    return Promise.using(this.ensureConnection(), function(connection) {
+      runner.connection = connection;
+
+      var sql = runner.builder.toSQL();
+
+      if (runner.builder._debug) {
```

Comment:
`sql` and if-statement should be moved to `run()` instead of `_run()` to avoid double `console.log` when a retry is called. And a better name for the function as well.